README.md

```diff
@@ -23,9 +23,10 @@ or run using `npx` or `exec`
 ## locktt
 
 **What locktt will do**:
- - looks for package-lock.json, npm-shrinkwrap.json and yarn.lock files
+ - looks for package-lock.json, npm-shrinkwrap.json, yarn.lock and pnpm-lock.yaml files
  - removes the `resolved` and `integrity` fields from the package-lock.json and/or npm-shrinkwrap.json files in case it finds them
  - replaces the `resolved` field from yarn.lock file adding the `--registry` value instead of the host and remove the `integrity` field
+ - replaces all hosts (in resolution field, in dependency references etc) with value specified in `--registry` and removes all the `integrity` fields from pnpm-lock.yaml file
```

Comment:
it is not part of the issue itself, but could you please adapt
```

### Restrictions
* Version **>=0.4.1**:
  * *NodeJS* >= 7.6.0
* Version **<=0.4.0**:
  * *NodeJS* >= 5.0.0
```
accordingly

Reply:
I didn't quite understand what needs to be changed here

Reply:
I mean the restrictions section from README.md file (see https://github.com/kiegroup/lock-treatment-tool#restrictions) is wrong for two reasons:

- we don't even have `0.4.0` version yet 
- we are supporting NodeJS version >=12

The content should be something like 
```
### Restrictions
*NodeJS* >= 12
```

Reply:
I will do it in this one